packages/data-transformation/fhir-to-csv/src/setupSnowflake/setupSnowflake.py

```
@@ -0,0 +1,130 @@
+import snowflake.connector
+import os
+import configparser
+import logging
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)